### PR TITLE
Add Edge event products example

### PR DESCRIPTION
**Pull request by rymorale**

### CODE REVIEW

File: using-mobile-extensions/adobe-analytics-edge/product-variable.md [hunk 1]
```diff
@@ -1,6 +1,123 @@
...
+Edge.sendEvent(experienceEvent, new EdgeCallback() {
+  @Override
+  public void onComplete(List<EdgeEventHandle> list) {
+    // handle the response
+  }
+});
+}
+```
+
+The alternative to using raw dictionaries for your XDM data is building a concrete class based on your XDM schema. For more information, see the Schema interface at https://aep-sdks.gitbook.io/docs/v/AEP-Edge-Docs/using-mobile-extensions/adobe-edge/edge-api-reference#public-classes, and the commerce example in [AEP SDK Sample App for Android](https://github.com/adobe/aepsdk-sample-app-android).
```
Comment (emdobrin) [inline using-mobile-extensions/adobe-analytics-edge/product-variable.md:75]: .. see the Schema interface in Edge API reference Public classes <link> and the ...

File: using-mobile-extensions/adobe-analytics-edge/product-variable.md [hunk 2]
```diff
@@ -1,6 +1,123 @@
...
+
+#### Example
+
+```swift
+// build a product map for each product
+let product1 = ["SKU":"SH100","name":"Running Shoes","priceTotal": 69.95, "quantity": 1] as [String : Any]
+let product2 = ["SKU":"SO100","name":"Running Socks","priceTotal": 29.99, "quantity": 10] as [String : Any]
+
+// build a product list and add the product maps
+let productListItems: [Dictionary<String, Any>] = [product1, product2]
```
Comment (emdobrin) [inline using-mobile-extensions/adobe-analytics-edge/product-variable.md:91]: this should be [[String: Any]]

File: using-mobile-extensions/adobe-analytics-edge/product-variable.md [hunk 3]
```diff
@@ -1,6 +1,123 @@
...
+// build an order, purchases, and commerce map
+let order = ["purchaseID":"1234567890","priceTotal":orderTotal,"currencyCode":"USD","payments":paymentsItem] as [String : Any]
+
+let purchases = ["value":1.0]
+
+let commerce = ["order":order,"purchases":purchases]
+
+// create an Experience Event and send it using the AEP Edge extension
+let xdmData =  ["commerce":commerce,"productListItems":productListItems,"eventType":"commerce.purchases"] as [String : Any]
+let event = ExperienceEvent(xdm: xdmData, datasetIdentifier: "60106c7230c86f1948613a91")
```
Comment (emdobrin) [inline using-mobile-extensions/adobe-analytics-edge/product-variable.md:110]: datasetIdentifier: "your-dataset-id"

File: using-mobile-extensions/adobe-analytics-edge/product-variable.md [hunk 4]
```diff
@@ -1,6 +1,123 @@
...
+Edge.sendEvent(experienceEvent: event)
+```
+
+The alternative to using raw dictionaries for your XDM data is building a concrete class based on your XDM schema. For more information, see the XDMSchema protocol at https://aep-sdks.gitbook.io/docs/v/AEP-Edge-Docs/using-mobile-extensions/adobe-edge/edge-api-reference#public-classes, and the commerce example in [AEP SDK Sample App for iOS](https://github.com/adobe/aepsdk-sample-app-ios/tree/main/Swift).
```
Comment (emdobrin) [inline using-mobile-extensions/adobe-analytics-edge/product-variable.md:114]: XDMSchema protocol in Edge API reference Public classes <link> and ..

**Review by emdobrin — COMMENTED** [review]
Looks good @rymorale , left some small suggestions

**Review by emdobrin — APPROVED** [review]

**Comment by praschetan** [comment]
Closing pending internal review of Adobe Analytics Edge extension release.

**Pull request closed without merging**